force-app/main/default/objects/ServiceSession__c/fields/OmittedContactIds__c.field-meta.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>OmittedContactIds__c</fullName>
+    <description>This field is used to track which contacts were admitted from the Attendance roster. This is a system field and should not be manually altered.</description>
```

Comment:
Do you mean omitted  instead of admitted here? If so: 

This field is used to track which contacts were omitted from the Attendance roster. This is a system field and shouldn't be manually altered.

Reply:
@jencrabtree, Can you please look at the change and approve it since you had this comment. I see that Jenny has fixed it.